src/ui-extensions.js

```diff
@@ -5,16 +5,19 @@
  * This should be used to define the corresponding onOpen
  * trigger in a Apps script connected to a spreadSheet.
  *
- * @param {object} e - onOpne event (unused)
+ * @param {object} e - onOpen event (unused)
  * @param {string} libraryName - Name the GS in the Spreadsheet setup.
  */
 function onOpen(e, libraryName) {
   const ui = SpreadsheetApp.getUi();
 
-  ui.createMenu('Metabse')
+  ui.createMenu('Metabase')
     .addItem('Initialize Metabase', `${libraryName}.initializeMetabase`)
     .addItem('Preview Query', `${libraryName}.queryPreviewInput`)
     .addItem('Delete User Data', `${libraryName}.deleteUserData`)
+    .addItem('Get All cards', `${libraryName}.getListOfCards`)
+    .addItem('Process Selected Cards', `${libraryName}.processSelected`)
```

Comment:
Could 'Create Report' be a better name? :)

Reply:
I do like that the name contains "selected" because that makes the connection to the check boxes quite clear. Maybe a combination like "Reports from selected cards" or "Selected Cards to Report".